chromium_src/BUILD.gn

```diff
@@ -8,6 +8,14 @@ if (is_win) {
   }
 }
 
+source_set("modules") {
+  sources = [
+    "modules/battery/battery_disabled_api_s.h",
```

Comment:
I think this source set should be 'blink' and match the path/class names. Then the patches aren't necessary